packages/core/src/server/service.ts

```diff
@@ -124,6 +131,33 @@ export class ServerService {
       }
     });
 
+    // TODO: Only register in development mode.
+    this.app.post("/admin/reload", async (req, res) => {
+      try {
+        const chainIdsQuery = req.query.chainIds ?? "31337";
+        if (typeof chainIdsQuery !== "string") {
+          res.status(400).end("chainIds must be a string");
+        }
+
+        // TODO: Enforce number.
+        const chainIds: number[] = [];
```

Comment:
Maybe you disagree, but I think it would be reasonable to only support one chain at a time (what would be the use case for many at once)?